microsoft/PowerToys
# [PT Run/FancyZones] Exclude Launcher from zonable windows improvement

## Summary of the Pull Request

**What is this about:**
Improve how PT Run is excluded from FancyZones

**What is include in the PR:** 
Added WS_POPUP to GWL_STYLE

Setting Window as ToolWindow wasn't an option
`System.InvalidOperationException: WindowStyle.None is the only valid value for WindowStyle when AllowsTransparency is true.`

**How does someone test / validate:** 
- Enable FancyZones option Override Windows Snap Hotkeys
- Open PT Run
- Pressing Win + Arrow PT Run should't be snapped into zones

## Quality Checklist

- [x] **Linked issue:** #2911

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
